code/modules/research/techweb/all_nodes.dm

```diff
@@ -2467,3 +2467,30 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	hidden = TRUE
 	experimental = TRUE
+
+/datum/techweb_node/mod_experimental
+	id = "mod_experimental"
+	display_name = "Experimental Modular Suits"
+	description = "Applications of experimentality when creating MODsuits have created these..."
+	prereq_ids = list("base")
+	design_ids = list(
+		"mod_disposal",
+		"mod_joint_torsion",
+		"mod_recycler",
+		"mod_shooting",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	hidden = TRUE
+	experimental = TRUE
+
+/datum/techweb_node/posisphere
+	id = "positronic_sphere"
+	display_name = "Experimental Spherical Positronic Brain"
+	description = "Compplaints of bitching posibrains have brought forward a new generation of pestering."
```

Comment:
```suggestion
	description = "Complaints of bitching posibrains have brought forward a new generation of pestering."
```

Reply:
alternatively if you take my suggestion on costs

```suggestion
	description = "Recent developments on cost-cutting measures have allowed us to cut positronic brain cubes into cheaper spheres. Unfortunately, it also allows them to move around the lab via rolling maneuvers."
```

Reply:
Alright I gotta admit you got some good writing. I'll use it as desc aswell